cfme/tests/infrastructure/test_providers.py

```diff
@@ -58,7 +58,7 @@ def test_add_cancelled_validation():
 def test_type_required_validation():
     """Test to validate type while adding a provider"""
     prov = InfraProvider()
-    with error.expected(FlashMessageException):
```

Comment:
Hmmmm, I'm not sure on this, I actually quite liked the older pattern, as it gave you an idea what you expected to fail, and that it was expected.